test/simulation/README.md

```diff
@@ -38,7 +38,7 @@ $ GO111MODULE=on go test -v
 - [ ] Peer disconnection should trigger orders removal to all his orders.
 
 ### Node State
-- [ ] Removed pair should trigger order removal for all the pair's orders, and removed from connected peers' order books.
+- [ ] Removed pair or swap client (lnd/raiden) unavailable should trigger order removal for all the pair's orders, and removed from connected peers' order books.
```

Comment:
Do we want to trigger order removals immediately when swap client becomes unavailable? @sangaman suggested to have a short timeout period before we do remove orders: https://github.com/ExchangeUnion/xud/issues/684#issuecomment-483110342

Reply:
No, we don't i agree on the timeout period, see https://github.com/ExchangeUnion/xud/issues/684#issuecomment-483209152. Clarified that.